app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -16,6 +16,15 @@
           i,
           j;
 
+      // page level custom dimension
+      $('meta[name="ga_customDimension"]').each(function(index, customDimension){
+          var dimensionId = $(customDimension).attr('data-id');
```

Comment:
Out of interest, does this still work if you remove the `$(..)`? I'd have thought so, as JQuery should already be passing through Jquery objects when you use `.each`. Doesn't much matter obviously.

One more thing: this code only seems to get called on page load? Does that mean that the custom dimensions can't be attached to virtual page views, and does that matter for our current use case? Maybe it's just a case of noting that limitation somewhere rather than fixing it.

Reply:
Regarding the `$(..)` I usually always use the variable around the jquery tags due to some instances it not being translated properly. 

Regarding the code only running on page load, this would be correct. For virtual page views, we have extended the virtual pageview functions already to allow passing custom dimensions through. ref: https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/assets/javascripts/analytics/_virtualPageViews.js#L23